app/views/agreements/detail.njk

```diff
@@ -62,7 +62,7 @@
       value: { html: agreementStatusTag(agreement.status) }
     },{
       key: { text: "Description" },
-      value: { html: agreement.description },
+      value: { html: agreement.description | striptags(true) | escape | nl2br },
```

Comment:
I think the `text` approach is better. I can’t really think why this was `html` in the first place. Possibly it was just copied from the one above.